### ISSUE TITLE
People icon

### ISSUE THREAD

**Issue by fredenocs** [Sycnex/Windows10Debloater#64]
Running the sysprep version on MDT environment.

I am not experiencing the peoples icon on the taskbar being removed?  Am I missing something?

**Comment by Sycnex** [comment]
Hmmm, in the past my script would just allow you to turn the People Icon off because otherwise it was locked by a registry key.

Can you confirm that you have the ability to turn it off 

Settings --> Personalization --> Taskbar --> People

**Comment by fredenocs** [comment]
Created new user on my test VM.

Yes I can turn off.  In my environment we want it off; along with the notifications for any all users logging in.

**Comment by fredenocs** [comment]
Wow, I just put 2 n 2 together.  Tested a quick theory.

My test deployments aren't activated version of Windows, while I test my settings.  Since I posted this, I created another Win10Ent1803 VM.  The Sysprep version is built into the MDT deployment.

Rebooted and created new user, people's icon on taskbar never removed.  I noticed you canNOT personalize Windows unless you're activated it.  I tested this theory.  So all settings are grayed out.

The machine I am using is a W101803, activated, course I ran the debloater on it when I created it; after not during MDT deployment.  New user people's icon doesn't appear.

So this will answer my question on disabling the notifications; I cannot unless I activate.
